admin/category_product_listing.php

```diff
@@ -711,30 +730,45 @@
               <?php
             }
 
-
-            $order_by = " ";
             switch ($_SESSION['categories_products_sort_order']) {
-              case (0):
-                $order_by = " ORDER BY p.products_sort_order, pd.products_name";
-                break;
-              case (1):
-                $order_by = " ORDER BY pd.products_name";
-                break;
-              case (2);
-                $order_by = " ORDER BY p.products_model";
-                break;
-              case (3);
-                $order_by = " ORDER BY p.products_quantity, pd.products_name";
-                break;
-              case (4);
-                $order_by = " ORDER BY p.products_quantity DESC, pd.products_name";
-                break;
-              case (5);
-                $order_by = " ORDER BY p.products_price_sorter, pd.products_name";
-                break;
-              case (6);
-                $order_by = " ORDER BY p.products_price_sorter DESC, pd.products_name";
-                break;
+                case (0):
+                    $order_by = " ORDER BY p.products_sort_order, pd.products_name";
+                    break;
+                case (1):
+                    $order_by = " ORDER BY pd.products_name";
+                    break;
+                case (2):
+                    $order_by = " ORDER BY p.products_model";
+                    break;
+                case (3):
+                    $order_by = " ORDER BY p.products_quantity, pd.products_name";
+                    break;
+                case (4):
+                    $order_by = " ORDER BY p.products_quantity DESC, pd.products_name";
+                    break;
+                case (5):
+                    $order_by = " ORDER BY p.products_price_sorter, pd.products_name";
+                    break;
+                case (6):
+                    $order_by = " ORDER BY p.products_price_sorter DESC, pd.products_name";
+                    break;
+                case (7)://steve added
```

Comment:
"Lose" the `//steve added` comments, please.

Reply:
oops

Reply:
Thanks, @torvista.